css/styles.css

```diff
@@ -0,0 +1,480 @@
+}
+
+/* Main-Header Section Ends */
+
+/* Form Main Begins */
+.sub-header {
+  padding: 0;
+  margin: auto;
+}
+
+.mini-main-header {
+  padding-top: 0;
+  text-align: center;
+}
+
+.mini-logo-container {
+  margin: 0 auto;
+  display: block;
+  width: 100%;
+}
+
+.logo-list {
+  margin-left: auto;
+  margin-right: auto;
+  list-style: none;
+  margin-bottom: 15px;
```

Comment:
- [ ] **[OPTIONAL]** You can use the shorthand for margins so you don't have to specify the different margins:
```
margin: 0 auto 15px auto;     /*That is top, right, bottom, left*/
```

Reply:
Done.